project/.travis/before_install_test.sh.twig

```diff
@@ -2,11 +2,16 @@
 set -ev
 
 if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then
-    mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp
-    echo "memory_limit=3072M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    PHP_INI_DIR="$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/"
+    XDEBUG_INI_FILE="$PHP_INI_DIR/xdebug.ini"
+    TRAVIS_INI_FILE="$PHP_INI_DIR/travis.ini"
+    if [ -f  "$XDEBUG_INI_FILE" ]; then
+        mv "$XDEBUG_INI_FILE" /tmp
+    fi
+    echo "memory_limit=3072M" >> "$TRAVIS_INI_FILE"
 
-    if [ ${TRAVIS_PHP_VERSION} '<' '7.0' ]; then
-        echo "extension=mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    if [ "$TRAVIS_PHP_VERSION" '<' '7.0' ]; then
```

Comment:
We don't, it's just to make shellcheck happy and male sure not to miss other potential warnings because of this one